### PR TITLE
Added Login button in navbar

### PR DESCRIPTION
**Pull request by MayankVir**
## What kind of change does this PR introduce?

UI enhancement

## What is the current behavior?

Currently, we don't have a separate login button. #4739 

## What is the new behavior?

Added the login button in the navbar

## Additional context

image

### CODE REVIEW

**Comment by prescience-data** [comment]
From what @MildTomato mentioned here https://github.com/supabase/supabase/issues/4739#issuecomment-1006160419 I believe he preferred to use `Sign in` as the nomenclature.

File: www/components/Nav/index.tsx [hunk 1]
```diff
@@ -243,10 +243,16 @@ const Nav = (props: Props) => {
...
                 <a href="https://github.com/supabase/supabase" target="_blank">
-                  <Button type="default" icon={<IconStar />}>
+                  <Button
+                    className="sbui-btn sbui-btn-text sbui-btn--tiny sbui-btn--text-align-center"
+                    icon={<IconStar />}
+                  >
                     Star us on GitHub
                   </Button>
                 </a>
+                <a href="https://app.supabase.io/" target="_blank">
```
Comment (prescience-data) [inline www/components/Nav/index.tsx:253]: What is the rationale for `target="_blank"` here? 🔎

File: www/components/Nav/index.tsx [hunk 2]
```diff
@@ -243,10 +243,16 @@ const Nav = (props: Props) => {
...
+                  <Button type="default">Log in</Button>
```
Comment (prescience-data) [inline www/components/Nav/index.tsx:254]: According to https://github.com/supabase/supabase/issues/4739#issuecomment-1006160419 should be `Sign in` to mirror the Supabase Studio label. 👍🏻

File: www/components/Nav/index.tsx (hunk 1 above)
Comment (MayankVir) [reply www/components/Nav/index.tsx:253]: No specific reasons for `target="_blank"`. We can remove it so the user should remain on the same page while signing in?

File: www/components/Nav/index.tsx (hunk 2 above)
Comment (MayankVir) [reply www/components/Nav/index.tsx:254]: Sure, I can change the nomenclature to `Sign in`  so that it matches with Studio Label.

**Comment by MayankVir** [comment]
Please review the recent commit if it needs any improvements further?

File: www/components/Nav/index.tsx [hunk 3]
```diff
@@ -243,10 +243,16 @@ const Nav = (props: Props) => {
...
+                  >
                     Star us on GitHub
                   </Button>
                 </a>
+                <a href="https://app.supabase.io/">
+                  <Button type="default">Sign in</Button>
+                </a>
                 <a href="https://app.supabase.io/">
                   <Button>Start your project</Button>
                 </a>
```
Comment (Isaiah-Hamilton) [inline www/components/Nav/index.tsx:258]: Shouldn't we use `Link` from Nextjs?

Comment (MayankVir) [reply www/components/Nav/index.tsx:258]: Yes, we should use but previously this component is using anchors for the other 2 links, so to maintain consistency I did not use Link. I can update that if that sounds good.

Comment (kiwicopple) [reply www/components/Nav/index.tsx:258]: Link is better for "internal" links (prevents full reload), and anchor tags for external

**Comment by kiwicopple** [comment]
looks great! Thanks for the contribution @MayankVir

**Comment by MayankVir** [comment]
Thanks @kiwicopple, this was my first contribution to Open source, and hope to contribute more. 😁

**Comment by MayankVir** [comment]
Hi @kiwicopple, All the checks have been passed but PR has not been merged yet.

**Review by kiwicopple — APPROVED** [review]

**Comment by kiwicopple** [comment]
> first contribution to Open source

🎉 please send an email to swag@supabase.io (link to this PR) and we can send you some swag!

**Pull request merged**